lib/Core/Executor.cpp

```diff
@@ -2454,6 +2454,9 @@ void Executor::executeInstruction(ExecutionState &state, KInstruction *ki) {
 void Executor::updateStates(ExecutionState *current) {
   if (searcher) {
     searcher->update(current, addedStates, removedStates);
+    searcher->update(0, continuedStates, pausedStates);
```

Comment:
If you can, it would be great if you can start using `c++11`.
* `0`-> `nullptr`